share/sources/uk.cambridge/source.yaml

```diff
@@ -13,7 +13,11 @@ configs:
     approved_sets: null
     emitted_type: CreativeWork
     property_list: []
-    type_map: {}
+    type_map:
+      Journal Article: article,
+      Book chapter: book,
+      Book or Book Chapter: book,
+      Conference Object: conferencepaper,
```

Comment:
YAML's kinda weird; these trailing commas are actually part of the string.

Reply:
Oops. This worked fine though?

Reply:
Was that when you had the brackets in? Apparently this:
```yaml
foo: {
  apples: bananas,
  potatoes: toes,
}
```
sees the commas as delimiters, but this
```yaml
foo:
  apples: bananas,
  potatoes: toes,
```
sees the commas as part of the strings.

Reply:
Oh possibly. That would explain it